README.md

```diff
@@ -140,6 +140,7 @@ Injects a fake request into an HTTP server.
     - `body` - alias for payload.
     - `rawPayload` - the raw payload as a Buffer.
     - `trailers` - an object containing the response trailers.
+    - `json` - a function that parses string to json.
```

Comment:
I would improve the description:
`json()` - parse the application/json response payload and returns an object. Throws if the content type is different